sam/auxtags.go

```diff
@@ -273,8 +279,8 @@ func ParseAux(text []byte) (Aux, error) {
 		return nil, fmt.Errorf("sam: invalid aux tag field: %q", text)
 	}
 	var t Tag
-	if copy(t[:], tf[0]) != 2 {
-		return nil, fmt.Errorf("sam: invalid aux tag: %q", tf[0])
+	if copy(t[:], text[:2]) != 2 {
```

Comment:
:) Yes, thanks.

The other option is 

```
aux, err := NewAux(Tag{text[0], text[1]}, value)
```

and drop the `var t Tag; copy(...`.